views/includes/page-templates/beneficial-owner-nature-of-control-types.html

```diff
@@ -14,7 +14,8 @@
   {% include "includes/inputs/nature-of-control-checkboxes/trustees-nature-of-control-types.html" %}
 {% endif %}
 
-{% if FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC and not entity_number %}
+{# if feature flag on, show following for all registration BOs and BO Individual on the update/remove journey#}
+{% if FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC and ((not entity_number) or isBeneficialOwnerIndividual) %}
```

Comment:
Why did you add this condition?
and ((not entity_number) or isBeneficialOwnerIndividual)

Just remove:
and not entity_number

Reply:
Mine are ready and we can merge them at the same time.

In any case, as long as you don't include ‘beneficial-owner-nature-of-control-jurisdictions.html’ in the view, they won't be displayed.

Reply:
ok, it makes sense to remove it, we are making the changes at the same time anyway. I'll update the html.

Reply:
updated